docs/windows.md

```diff
@@ -0,0 +1,34 @@
+# Windows Installation
+To setup and install PaSh on a Windows machine, `Windows Subsystem for Linux (WSL)` needs to be installed.
+
+## What is WSL
+The Windows Subsystem for Linux lets developers run a GNU/Linux environment 
+-- including most command-line tools, utilities, and applications -- 
+directly on Windows, unmodified, without the overhead of a traditional virtual 
+machine or dualboot setup.
+
+## Setup WSL
+
+- Open PowerShell as Administrator and run:
+
+    `dism.exe /online /enable-feature /featurename:Microsoft-Windows-Subsystem-Linux /all /norestart`
+
+- Enable Linux Filesystem for Windows
+
+    `dism.exe /online /enable-feature /featurename:VirtualMachinePlatform /all /norestart`
+
+- Install the distro of your choice:
```

Comment:
Maybe only mention distros we have tried/supported by PaSh for now?

Reply:
Fixed!